modules/howtos/examples/subdoc.js

```diff
@@ -152,13 +152,23 @@ async function go() {
   // end::mutate-tl-arrayappend-multibad[]
 
   console.log('mutate-createpaths')
-  // tag::mutate-createpaths[]
-  await collection.mutateIn('customer123', [
-    couchbase.MutateInSpec.arrayAppend('some.path', 'Hello World', {
-      createPath: true,
-    }),
-  ])
-  // end::mutate-createpaths[]
+  try {
```

Comment:
this commit looks fine, though the subject looks more like "deal with JSCBC-1045" than Minor Fixes?